_build_cfg.yml

```diff
@@ -615,8 +615,8 @@ Topics:
     Topics:
       - Name: Overview
         File: index
-      - Name: Using S2I images
-        File: customization
+      - Name: Customizing S2I images
```

Comment:
@ahardin-rh Blast from the past! But...

s/images/Images/

I think it could also be at the end of the list of images, instead of the first?